tools/schema_tools/gen_output.py

```diff
@@ -3,6 +3,7 @@
 import MySQLdb as mysqldb
 import os
 from socket import gethostname
+from lsst.sims.utils import photo_m5
```

Comment:
I was trying to remember what name we came up with...I knew it was something to say we weren't using the full sed.  Putting "sed" in the name seems to me to imply it's using a full sed.  I have no overwhelming preference:
a) m5_flat_sed
b) photometry_m5
c) ?